js/mapModel.js

```diff
@@ -31,24 +32,17 @@ class mapModel{
     }
 
     getTile(X, Y){
+        if( this.mapHash[X+','+Y] === undefined ) {
```

Comment:
This is a conversation for another time, but I do want to go over this logic to automatically add default, blank tiles. I think this is fine, and that there's no harm done adding/storing these default tiles just because the hero looked at them. But, maybe it's wasteful or mildly harmful.

Reply:
The idea is that any tiles that don't contain anything important are automatically Meadow tiles.  While we don't need to store them the map needs to stay revealed for the hero.  While I don't know the details of the view file it may require more logic on that end if we delete these old tiles, this could also create an increase in bugs.  Secondly, if the hero ever goes back to one of these spots we need to generate the tile again.  Since memory isn't an issue here and the retrieval time of any tile is O(1) I wouldn't suggest changing this logic unless we find some glaring bug because of it.